Source/WebCore/css/mediaControls.css

```diff
@@ -177,6 +177,70 @@ video::cue {
     background-color: rgba(0, 0, 0, 0.8);
 }
 
+video::cue(.white) { 
+    color: rgba(255,255,255,1);
```

Comment:
Two minor nits: 
- none of these colors has alpha so you could use `rgb(...)` instead of `rgba(..., 1)`
- WebKit style is to put a space between comma separated values, so this should be `rgb(255, 255, 255)`

Reply:
My own workflow for followup patches is:
- `git commit --amend path/to/new/changes`
- `git webkit pr --overwrite`